.gitmodules

```diff
@@ -1,8 +1,5 @@
-[submodule "its/sources"]
-	path = its/sources
-	url = https://github.com/SonarCommunity/python-test-sources.git
-[submodule "its/sources_extended"]
-	path = its/sources_extended
+[submodule "its/sources_ruling"]
+	path = its/sources_ruling
 	url = https://github.com/SonarCommunity/python-test-sources.git
 	branch = extended
```

Comment:
I'm a bit confused here, how are we actually retaining the original ruling sources (from `master`)? It seems the only submodule defined is for the `extended` branch and that branch hasn't changed?
I'd have expected to see the `master-temp` branch here?

Reply:
On the python-test-sources side, I've essentially merged the two branches into `master-temp`, which is checked out in the `its/sources_ruling` submodule. 
According to [this StackOverflow answer](https://stackoverflow.com/questions/5033441/where-does-git-store-the-sha1-of-the-commit-for-a-submodule), git keeps track of the hash of the submodules directly in its database. That's probably the reason for the following diff
![image](https://github.com/user-attachments/assets/96da30b0-b31a-4e1c-ba3c-9b5df14ffcee)

Reply:
I've changed the .gitmodules and added `shallow = true` (see [here](https://git-scm.com/docs/gitmodules#Documentation/gitmodules.txt-submoduleltnamegtshallow) for documentation)